api/graphql/schema.graphql

```diff
@@ -165,10 +162,189 @@ input UpdateUserInput {
     preferredLang: Language @rename(attribute: "preferred_lang")
 }
 
+input LocalizedStringInput {
+    en: String
+    fr: String
+}
+
+input CreateUserBelongsTo {
+    connect: ID
+}
+
+input CreatePoolBelongsTo {
+    connect: ID
+}
+
+input CreateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    level: Int!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input CreateClassificationBelongsToMany {
+    create: [CreateClassificationInput!]
+    connect: [ID!]
+}
+
+input CreateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateCmoAssetBelongsToMany {
+    create: [CreateCmoAssetInput!]
+    connect: [ID!]
+}
+
+input CreateOperationalRequirementInput {
+    key: String
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateOperationalRequirementBelongsToMany {
+    create: [CreateOperationalRequirementInput!]
+    connect: [ID!]
+}
+
+input CreatePoolCandidateInput {
+    pool: CreatePoolBelongsTo @belongsTo
+    user: CreateUserBelongsTo @belongsTo
+    cmoIdentifier: ID @rename(attribute: "cmo_identifier")
+    expiryDate: Date
+    isWoman: Boolean @rename(attribute: "is_woman")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    acceptedOperationalRequirements: [OperationalRequirement] @belongsToMany
+    expectedSalary: [SalaryRange] @rename(attribute: "expected_salary")
+    expectedClassifications: [Classification] @belongsToMany
+    cmoAssets: [CmoAsset] @belongsToMany
+    status: PoolCandidateStatus
+}
+
+input CreatePoolCandidateHasMany {
+    create: [CreatePoolCandidateInput!]
+}
+
+input CreatePoolInput {
+    owner: CreateUserBelongsTo @rename(attribute: "user")
```

Comment:
owner for CreatePoolInput should be !required. For this one, I would think to allow only the connect operation.

Reply:
It still seems weird to me to allow creating a new user and assigning it to the Pool owner at the same time... Am I thinking about the API to restrictively?
Once we have authentication, we may want to set the current user as the owner directly (with the [inject](https://lighthouse-php.com/master/api-reference/directives.html#inject) directive) and this'll become a moot point? So don't worry about it.